Catalog metrics port

eklesel opened the pull request.

c-thiel commented:
@eklesel should it be OK for you to have a configurable monitor instead of the old labels?
Similar to: https://github.com/airflow-helm/charts/blob/732a0f0bdbeb420159807d3d5f858e614c6656ec/charts/airflow/values.yaml#L2390

eklesel commented:
@c-thiel I've added a value now to switch these on and off, defaults to on.

I've also updated the README using the `helm-docs` command, but I noticed that there were some more changes than just mine since the last time this was run. I've removed the other additions to make sure only mine are committed and not mix up commits.

eklesel commented on charts/lakekeeper/values.yaml:
```diff
@@ -297,6 +297,10 @@ catalog:
 
+  prometheusMetrics:
+    # -- Enables the Metrics port and adds the relevant Prometheus labels
+    enabled: true
```
Variable renamed and set to default to false.
